google-cloud-spanner/src/main/java/com/google/cloud/spanner/MultiplexedSessionDatabaseClient.java

```diff
@@ -164,6 +172,13 @@ public void close() {
   /** The current multiplexed session that is used by this client. */
   private final AtomicReference<ApiFuture<SessionReference>> multiplexedSessionReference;
 
+  @VisibleForTesting
```

Comment:
nit: the variable is `private`, so it is not visible (also not for testing), so this annotation should be removed

Reply:
Done